server/router.js

```diff
@@ -1,11 +1,17 @@
 const mem = require("./controllers/memories-controller")
 const com = require("./controllers/comments-controller")
+const admin = require("./controllers/admin-controller")
```

Comment:
This looks good!
Do you need an endpoint to get an individual memory?
like GET memories/:id